Content/Forest/Safekeeper/UndeadNPC.cs

```
@@ -27,18 +25,27 @@ public class UndeadNPC : GlobalNPC
 		NPCID.GiantCursedSkull, NPCID.Frankenstein, NPCID.DD2SkeletonT1, NPCID.DD2SkeletonT3, NPCID.Poltergeist, NPCID.Wraith, NPCID.FloatyGross, NPCID.Mummy, 
 		NPCID.BloodMummy, NPCID.DarkMummy, NPCID.LightMummy, NPCID.Ghost];
 
-	private static readonly HashSet<NPC> ToDraw = [];
 	private static bool TrackingGore;
 
-	/// <summary> Decreases by <see cref="DecayRate"/>. </summary>
-	private float _decayTime = 1;
-
-	public override bool InstancePerEntity => true;
-
 	internal static bool AddCustomUndead(params object[] args)
 	{
-		if (args.Length == 2 && args[1] is int customType) //Context, undead type
-			return UndeadTypes.Add(customType);
+		switch (args.Length)
```

A couple things - usually, I'll pass `args[1..]` to the receiving method for a Mod.Call since that removes the then-useless `context` parameter, really not a big change but it's a mild convenience.
More importantly, I think it's important to throw verbosely if a Mod.Call fails - the way this is set up atm would silently fail which just adds more dev time. I'd prefer explicitly throwing per value; i.e. in case 2 here `return new ArgumentException("Argument 1 is not int customType!");` so other devs can more easily tell what they're doing wrong.